src/phel/core.phel

```diff
@@ -1375,6 +1375,17 @@ returns 1. If `xs` has one value, returns the reciprocal of x."
   (php/print "\n")
   nil)
 
+(defn format
+  "Returns a string produced according to the formatting string fmt."
```

Comment:
Could we reference the PHP Doc here? Like this:

"Returns a formatted string. See PHP's `sprintf` for more information`"